### DISCUSSION TITLE
Wimlib install help

### DISCUSSION THREAD

**Discussion by ElHazard713** [Homebrew/discussions#5735] [Getting started]
### Output of `brew config`

```shell
Homebrew Ruby
```

### Output of `brew doctor`

```shell
Warning: You have unlinked kegs in your Cellar.
Leaving kegs unlinked can lead to build-trouble and cause formulae that depend on
those kegs to fail to run properly once built. Run `brew link` on these:
  openssl@3

Warning: You are using macOS 11.
We (and Apple) do not provide support for this old version.
It is expected behaviour that some formulae will fail to build in this old version.
It is expected behaviour that Homebrew will be buggy and slow.
Do not create any issues about this on Homebrew's GitHub repositories.
Do not create any issues even if you think this message is unrelated.
Any opened issues will be immediately closed without response.
Do not ask for help from Homebrew or its maintainers on social media.
You may ask for help in Homebrew's discussions but are unlikely to receive a response.
Try to figure out the problem yourself and submit a fix as a pull request.
We will review it but may or may not acce
```

### Description of issue

*updated

95-test_external_pyca.t ................. skipped: No external tests in this configuration
95-test_external_tlsfuzzer.t ............ skipped: No external tests in this configuration
99-test_ecstress.t ...................... ok
90-test_store.t ......................... ok

Test Summary Report
-------------------
82-test_ocsp_cert_chain.t             (Wstat: 256 Tests: 3 Failed: 1)
  Failed test:  3
  Non-zero exit status: 1
Files=323, Tests=3770, 477 wallclock secs (22.80 usr  3.13 sys + 767.55 cusr 274.93 csys = 1068.41 CPU)
Result: FAIL
make[2]: *** [run_tests] Error 1
make[1]: *** [_tests] Error 2
make: *** [tests] Error 2

Do not report this issue to Homebrew/brew or Homebrew/homebrew-core!

Error: You are using macOS 11.
We (and Apple) do not provide support for this old version.
It is expected behaviour that some formulae will fail to build in this old version.
It is expected behaviour that Homebrew will be buggy and slow.
Do not create any issues about this on Homebrew's GitHub repositories.
Do not create any issues even if you think this message is unrelated.
Any opened issues will be immediately closed without response.
Do not ask for help from Homebrew or its maintainers on social media.
You may ask for help in Homebrew's discussions but are unlikely to receive a response.
Try to figure out the problem yourself and submit a fix as a pull request.
We will review it but may or may not accept it.

--Installing wimlib trying to get access to the "wimlib-imagex split command". Been about 30 minutes and its stuck/not moving from the "make" line. Is it normal for it to take awhile to download or is there another way to accomplish making a bootable usb? Thanks in advance!

**Comment by p-linnane** [comment]
As explained in the warnings, you are on an unsupported OS so you are building it from source. You'll need to debug the build failure with upstream.
